server/channels/jobs/product_notices/worker.go

```diff
@@ -9,25 +9,25 @@ import (
 	"github.com/mattermost/mattermost/server/v8/channels/jobs"
 )
 
-const jobName = "ProductNotices"
-
 type AppIface interface {
 	UpdateProductNotices() *model.AppError
 }
 
 func MakeWorker(jobServer *jobs.JobServer, app AppIface) model.Worker {
+	const workerName = "ProductNotices"
+
```

Comment:
nit: This only used once, maybe we can use inline?

PS: There are several occurrences like this, my comment applies to others as well.

Reply:
I'm 2/5 that I like the style as it is, as it always the reader to quickly spot the name of a worker. The `workerName' is now attached to every log message and hence it should be visible.

Reply:
I see, makes sense.